server/src/main/java/com/hedera/block/server/persistence/storage/write/NoOpBlockWriter.java

```diff
@@ -32,27 +30,27 @@
  * designed to isolate the Producer and Mediator components from storage implementation during testing while still
  * providing metrics and logging for troubleshooting.
  */
-public class NoOpBlockWriter implements BlockWriter<List<BlockItem>> {
-
-    private final MetricsService metricsService;
+public class NoOpBlockWriter implements BlockWriter<List<BlockItemUnparsed>> {
 
     /**
      * Creates a new NoOpBlockWriter instance for testing and troubleshooting only.
      *
      * @param blockNodeContext the block node context
      */
     public NoOpBlockWriter(BlockNodeContext blockNodeContext) {
-        this.metricsService = blockNodeContext.metricsService();
         System.getLogger(getClass().getName()).log(INFO, "Using " + getClass().getSimpleName());
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public Optional<List<BlockItem>> write(@NonNull List<BlockItem> blockItems) throws IOException {
+    public Optional<List<BlockItemUnparsed>> write(@NonNull List<BlockItemUnparsed> blockItems) throws IOException {
         if (blockItems.getLast().hasBlockProof()) {
-            metricsService.get(BlocksPersisted).increment();
```

Comment:
Why remove the metrics service?

Reply:
This is the NoOpBlockWriter. Tallying the number of blocks persisted when we're not actually doing any writing is misleading